packages/fxa-settings/src/pages/Signup/Confirm/index.tsx

```diff
@@ -2,28 +2,55 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React, { useState } from 'react';
-import { logViewEvent, usePageViewEvent } from '../../../lib/metrics';
-import { RouteComponentProps } from '@reach/router';
+import React, { useCallback, useEffect, useState } from 'react';
+import {
+  logErrorEvent,
+  logViewEvent,
+  usePageViewEvent,
+} from '../../../lib/metrics';
+import { RouteComponentProps, useNavigate } from '@reach/router';
 import ConfirmWithLink, {
   ConfirmWithLinkPageStrings,
 } from '../../../components/ConfirmWithLink';
 import { REACT_ENTRYPOINT } from '../../../constants';
 import { ResendStatus } from '../../../lib/types';
+import AppLayout from 'fxa-settings/src/components/AppLayout';
+import { useAccount, useInterval } from 'fxa-settings/src/models';
+import { AuthUiErrors } from 'fxa-settings/src/lib/auth-errors/auth-errors';
+import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
+
+export const viewName = 'confirm';
+const POLLING_INTERVAL_MS = 2000;
+export const NAVIGATE_TIMEOUT = 2000;
```

Comment:
nit (and only because I see the two time based consts side-by-side): the line above has a `_MS` suffix. Not sure if we want to add that to this as well. 

Although a true nit since it doesn't seem like we consistently use that suffix:
```sh
git grep -I "[A-Z_]*_MS" packages | wc -l # 119

git grep -EIn "\w+_TIMEOUT" packages | wc -l # 61
git grep -EIn "\w+_TIMEOUT" packages | grep -v "_TIMEOUT_MS" | wc -l # 51
```